cms/templatetags/cms_tags.py

```diff
@@ -422,7 +421,9 @@ def render_extra_menu_items(context, obj, template='cms/toolbar/dragitem_extra_m
 
     if not items:
         return ''
-    return template.render(Context({'items': items}))
+    with context.push():
+        context.update({'items': items})
+        return template.render(context)
```

Comment:
why not just do return `template.render({'items': items})`. We were not passing the full context before and I'd like to avoid doing so (for performance reasons) unless necessary.